### ISSUE TITLE
Multiselect preview

### ISSUE THREAD

**Issue by lukaszjablonski** [QL-Win/QuickLook#645]
When multiple files are selected in the File Explorer while navigating with arrows/mouse scroll Quick.Look could show previews of only selected files.

**Comment by schroef** [comment]
Yes i miss this feature, coming from osx and being able to view multiple files is really nice

**Comment by emako** [comment]
There is no planned support.

Maybe there will be in the future
